src/components/toolbar/__tests__/__snapshots__/toolbarFieldRangedMonthly.test.js.snap

```diff
@@ -259,13 +259,13 @@ exports[`ToolbarFieldRangedMonthly Component should handle updating granularity
       {
         "filter": "beginning",
         "type": "SET_QUERY",
-        "value": "2019-07-01T00:00:00.000Z",
+        "value": "2018-08-01T00:00:00.000Z",
         "viewId": undefined,
       },
       {
         "filter": "ending",
         "type": "SET_QUERY",
-        "value": "2019-07-31T23:59:59.999Z",
+        "value": "2018-08-31T23:59:59.999Z",
```

Comment:
required: confirm this result against the previous result. we moved to using shift, pop in the unit test to help with consistency

Reply:
can confirm moving back to shift brings the og result. working as intended

![Screenshot 2025-06-13 at 2 13 20 PM](https://github.com/user-attachments/assets/312ce34d-5ddc-442a-9e53-8d61b84a3373)